### PR TITLE
Implement going to the song results for each category

### PR DESCRIPTION
**Pull request by skywalkerdude**

### CODE REVIEW

File: Hymns/Browse/BrowseResultsListView.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import SwiftUI
+import UIKit
```
Comment (finben) [inline Hymns/Browse/BrowseResultsListView.swift:2]: Why do you need UIKit?

Comment (skywalkerdude) [reply Hymns/Browse/BrowseResultsListView.swift]: removed!

File: Hymns/Browse/BrowseResultsListView.swift [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+struct BrowseResultsListView: View {
+
+    @Environment(\.presentationMode) var presentationMode
+    let viewModel: BrowseResultsListViewModel
```
Comment (finben) [inline Hymns/Browse/BrowseResultsListView.swift:6]: Make this a private let and add initializer as we normally do? I have already done this and the UIKit and can just push them to your branch if you agree?

Comment (skywalkerdude) [reply Hymns/Browse/BrowseResultsListView.swift:6]: Sure, sounds good!. I already removed the UiKit, but you can push the private let + initializer

**Pull request merged**